### DISCUSSION TITLE
stale bot

### DISCUSSION THREAD

**Discussion by xxcriticxx** [MajkiIT/polish-ads-filter#19052] [Ideas]
@MajkiIT / @hawkeye116477 / @krystian3w 

https://github.com/marketplace/stale

**Comment by krystian3w** [comment]
Bot musiałby umieć wołać Majkiego np. minęło 14 dni, czy problem jest aktualny `MajkiIT`? A po repozytorium AdGuard raczej nie widać tego, po prostu wątek po 30 dniach zamyka się i może umknąć nawet sztabowi co w 10 osób rozwijają filtry AdGuard.

**Reply by hawkeye116477** [reply]
Ale przecież jest w opisie napisane 
> optionally post a comment to notify contributors that the Issue or Pull Request will be closed.

PS: Jest jeszcze takie coś https://github.com/actions/stale oparte o GH Actions. W odróżnieniu od aplikacji nie wymaga instalacji przez admina, wystarczy tylko dodać odpowiedni wpis do pliku :smile:.

**Comment by xxcriticxx** [comment]
@hawkeye116477 pierwsza opcja mi sie lepiej podoba

**Reply by hawkeye116477** [reply]
A jakie masz argumenty za/przeciw?

Bo ja powiem tyle, że bot oparty o GitHub Actions jest bardziej ekologiczny, bo cały czas nie reaguje na zgłoszenia, a jedynie raz na jakiś czas, tzn. np. można ustawić, by uruchamiał się co 2 dni albo codziennie (cron). Oprócz tego ma też nieco więcej opcji => https://github.com/actions/stale#list-of-options.

A tutaj raczej też nie ma aż tak wielkiego ruchu, by bot musiał na bieżąco reagować.

------------------------------------

Aktualizacja:

Znalazłem argumenty przeciw rozwiązaniu nr 1:

[probot/stale#261](https://github.com/probot/stale/issues/261)

[probot/stale#278](https://github.com/probot/stale/issues/278)

[probot/stale#286](https://github.com/probot/stale/issues/286)

[probot/stale#300](https://github.com/probot/stale/issues/300)

 [probot/stale#306](https://github.com/probot/stale/issues/306)

[probot/stale#307](https://github.com/probot/stale/issues/307)

[probot/stale#308](https://github.com/probot/stale/issues/308)
